src/modules/Common/containers/Layout.tsx

```diff
@@ -29,7 +29,7 @@ const Layout = ({
   const { t } = useTranslation();
   return (
     <>
-      <CommonHead title={title} description={desc} twitterCard="/images/twitter-card.jpg" />
+      <CommonHead title={title} description={desc} image="/images/open-graph-image.png" />
```

Comment:
> If OpenGraph does not mandate the name, I'd be in favour of using `logo-cover` or some equivalent as the filename, instead of using the protocol as a name 🙂